### ISSUE TITLE
Increase time entries loading capabilities

### ISSUE THREAD

**Issue by pzljanb** [Puzzlepart/did#1220]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Give user the capability to select what part of time enties to be loaded before loading is performed.

### Description
Add a "Search row" abowe time entries table with the following Columns:

_Simple solution_
| PERIOD | EMPLOYEE | 
|----------|-------------|
| Select    | Select         |

**Where selection:**
- Period = THIS MONTH, LAST MONTH, THIS YEAR, LAST YEAR, TOTAL
- Employee = Selectable list with users with entries related to the current project

_Advanced solution:_
| START |  END  | EMPLOYEE | 
|--------|--------|-------------|
| Select | Select | Select         |

**Where selections is based on real data from project, like:**
- Start = Year-Week  on the format "YEAR-WW"
- Stopp = Year-Week  on the format "YEAR-WW"
- Employee = Selectable list with users with entries related to the current project

**Comment by olemp** [comment]
@pzljanb @okms 

Skal vi få til noe ala dette?

![image](https://github.com/user-attachments/assets/b0ebc971-fa19-4f33-af81-2932eafaad2d)
